react/src/reactTextMask.js

```diff
@@ -1,41 +1,26 @@
-import React, {PropTypes} from 'react'
-import createTextMaskInputElement from '../../core/src/createTextMaskInputElement'
-
-export const MaskedInput = React.createClass({
-  propTypes: {
-    mask: PropTypes.oneOfType([
-      PropTypes.array,
-      PropTypes.func,
-      PropTypes.bool,
-      PropTypes.shape({
-        mask: PropTypes.oneOfType([PropTypes.array, PropTypes.func]),
-        pipe: PropTypes.func
-      })
-    ]).isRequired,
-    guide: PropTypes.bool,
-    value: PropTypes.oneOfType([PropTypes.string, PropTypes.number]),
-    pipe: PropTypes.func,
-    placeholderChar: PropTypes.string,
-    keepCharPositions: PropTypes.bool,
-    showMask: PropTypes.bool
-  },
-
-  createTextMaskInputElement,
```

Comment:
ok.. there is a reason for it being there though.. it is to aid testing - so that you can test the function exists and is a `function`.. i agree it is a _little_ pointless and the component works just the same without it.. I'm unsure if I agree with removing it.. it is up for debate..

It tests that the `createTextMaskInputElement` function is imported correctly into the react component.  I can see that it is "testing for the sake of testing", and perhaps not needed...